src/main/features/directions-questionnaire/views/hearing-location.njk

```diff
@@ -0,0 +1,59 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, submitButton %}
+
+{% set heading = 'Choose a hearing location' %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <p>Tell us your preferred location for a hearing, in case the claim goes to one.</p>
+      <p>We’ll use the location you choose unless the claimant has an exceptional reason to object, for example accessibility needs.</p>
+      <p>You can choose any location - Guildford County Court is nearest to the home address you gave us.</p>
```

Comment:
Please use translation `t('....')` for above three

Reply:
njk removed as it is not required for this story